eclass-writing/text.xml

```diff
@@ -47,11 +47,23 @@ Roughly speaking, there are three kinds of eclass:
 <title>Adding and Updating Eclasses</title>
 <body>
 
+<note>
+Given that updating eclasses will invalidate the cache of all consumer ebuilds,
+it is good etiquette to ping other developers in e.g. <c>#gentoo-dev</c> on IRC
+or informally determine if there are other similar changes (mainly documentation)
+which should be pushed at the same time in order to avoid unnecessary cache regeneration
+within a few hours or days of each other.
+
+The same applies to your own work <d/> please prepare all of your commits and
+batch push them at once where possible.
+</note>
+
 <p>
 Before committing a new eclass to the tree, it must be emailed to the gentoo-dev
-mailing list with a justification and a proposed implementation. Do not skip
+mailing list with a justification, and a proposed implementation. Do not skip
```

Comment:
These are only two terms, so I don't think the comma is right.

Reply:
Was too sleepy. Absolutely.